pkg/codegen/docs/gen.go

```diff
@@ -469,14 +485,16 @@ func (mod *modContext) genConstructorTS(r *schema.Resource, argsOptional bool) [
 				Name: argsType,
 				Link: docLangHelper.GetDocLinkForResourceType(mod.pkg.Name, modName, argsType),
 			},
+			Comment: ctorArgsArgComment,
 		},
 		{
 			Name:         "opts",
 			OptionalFlag: "?",
 			Type: propertyType{
-				Name: "CustomResourceOptions",
+				Name: "pulumi.CustomResourceOptions",
```

Comment:
Is there a reason to add the `pulumi.` back here?

Reply:
Nothing in particular, just to make it clearer that the type `CustomResourceOptions` comes from the `pulumi` package/namespace/module, rather than whatever the current resource provider is. (e.g. not `aws.CustomResourceOptions`.) But if you feel strongly about it, happy to change it back.

Reply:
Yeah we dropped those prefixes intentionally in favor of not showing namespaces for types. FWIW, I, too, felt the same way as you did and had those prefixes in the beginning. :)

Reply:
We did decide collectively to remove these, but as I recall we didn't spend a lot of time thinking about it (I believe the intent was to conserve space), and looking at them now, they do make things clearer. I'm personally okay with putting them back in, but if others would prefer to save that kind of change for another PR, we could do that, too. A tooltip or hover (if we did want to handle it in another PR) could be a good way to strike a balance.